lib/Ix/DBIC/ResultSet.pm

```diff
@@ -515,6 +516,15 @@ sub _ix_wash_rows ($self, $rows) {
     for my $key ($by_type{boolean}->@*) {
       $row->{$key} = $row->{$key} ? $true : $false if defined $row->{$key};
     }
+
+    for my $key ($by_type{datetime}->@*) {
+      if ($row->{$key}) {
+        # Doesn't already look like an RFC3339 Zulu date?
+        if ($row->{$key} !~ /Z/) {
+          $row->{$key} = parsepgdate($row->{$key});
+        }
+      }
+    }
```

Comment:
I thought about the defaults last night, too.  I think that for now we should assume that defaults are always valid. In the future, maybe we'll want to validate them, and that will require some more significant rejiggering, but for now we can just say "don't screw up your own defaults, programmer" and I think it's okay. Agreed?